### PR TITLE
Handle GL_MAX_VIEWS_OVR in glGetInteger

### PR DESCRIPTION
**Pull request by pmuetschard**

### CODE REVIEW

File: gapis/api/gles/api/state_queries.api [hunk 1]
```diff
@@ -1006,6 +1006,10 @@ sub void GetStateVariable!T(GLenum name, bool isIndexed, GLuint index, T* v) {
     case GL_MAX_VERTEX_UNIFORM_VECTORS: {
       v[0] = as!T(ctx.Constants.MaxVertexUniformVectors)
     }
+    @if(Extension.GL_OVR_multiview)
+    case GL_MAX_VIEWS_OVR: {
+      write(v[0:1])
```
Comment (ben-clayton) [inline gapis/api/gles/api/state_queries.api]: If we don't have a constant for this that can be assigned out, please add a TODO to add one.

Comment (pmuetschard) [reply gapis/api/gles/api/state_queries.api]: I've added the constant. PTAL.

File: gapis/api/gles/helpers.go [hunk 2]
```diff
@@ -175,6 +175,7 @@ func DefaultConstants30() Constants {
 		MaxTransformFeedbackSeparateAttribs:       4,
 		MaxTransformFeedbackSeparateComponents:    4,
 		MaxTextureMaxAnisotropyExt:                2.0,
+		MaxViewsExt:                               2,
```
Comment (ben-clayton) [inline gapis/api/gles/helpers.go]: Maybe this should default to 1?

Comment (pmuetschard) [reply gapis/api/gles/helpers.go]: No, the minimum is 2.

**Pull request merged**